### ISSUE TITLE
[<UICollectionView> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key ibCalendarDataSource.

### ISSUE THREAD

**Issue by apcro** [patchthecode/JTAppleCalendar#1091]
**(Required) Version Number:**
8.0.0

## Description
After upgrading the version from 7.1.7 to 8.0.0 and renaming the required classes (as per the migration guide), a working calendar setup now no longer works.

## Steps To Reproduce
Update from v7.1.7 to v8.0.0 and rename necessary classes (including Storyboard references)

## Expected Behavior
Calendar works as before.

## Additional Context
Prior to the update the implementation was working and was being used in production builds.

v8.0.1 is not available through `pod update`, and forcing the version causes an error. `pod` is up to date, ios target is set to `11.0`

**Comment by DanijelHuis** [comment]
I got the same error, for some reason, Interface Builder doesn't set module correctly after I renamed JTAppleCalendarView to JTACMonthView. I am using xibs. I manually wrote JTAppleCalendar for module and it worked.

![before](https://user-images.githubusercontent.com/5382135/61301222-c66da500-a7e3-11e9-9f58-77befe9b099a.png)

![after](https://user-images.githubusercontent.com/5382135/61301231-cc638600-a7e3-11e9-8554-be3c3f7ee399.png)
